service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -29,25 +34,34 @@
 
     private final HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration;
     private final CafcassLookupConfiguration cafcassLookupConfiguration;
+    private final LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration;
     private final HmctsEmailContentProvider hmctsEmailContentProvider;
     private final CafcassEmailContentProvider cafcassEmailContentProvider;
+    private final CafcassEmailContentProviderSDOIssued cafcassEmailContentProviderSDOIssued;
     private final GatekeeperEmailContentProvider gatekeeperEmailContentProvider;
+    private final LocalAuthorityEmailContentProvider localAuthorityEmailContentProvider;
     private final NotificationClient notificationClient;
     private final Logger logger = LoggerFactory.getLogger(getClass());
 
     @Autowired
     public NotificationHandler(HmctsCourtLookupConfiguration hmctsCourtLookupConfiguration,
                                CafcassLookupConfiguration cafcassLookupConfiguration,
+                               LocalAuthorityEmailLookupConfiguration localAuthorityEmailLookupConfiguration,
                                NotificationClient notificationClient,
                                HmctsEmailContentProvider hmctsEmailContentProvider,
                                CafcassEmailContentProvider cafcassEmailContentProvider,
-                               GatekeeperEmailContentProvider gatekeeperEmailContentProvider) {
+                               CafcassEmailContentProviderSDOIssued cafcassEmailContentProviderSDOIssued,
+                               GatekeeperEmailContentProvider gatekeeperEmailContentProvider,
+                               LocalAuthorityEmailContentProvider localAuthorityEmailContentProvider) {
         this.hmctsCourtLookupConfiguration = hmctsCourtLookupConfiguration;
         this.cafcassLookupConfiguration = cafcassLookupConfiguration;
+        this.localAuthorityEmailLookupConfiguration = localAuthorityEmailLookupConfiguration;
         this.notificationClient = notificationClient;
         this.hmctsEmailContentProvider = hmctsEmailContentProvider;
         this.cafcassEmailContentProvider = cafcassEmailContentProvider;
+        this.cafcassEmailContentProviderSDOIssued = cafcassEmailContentProviderSDOIssued;
```

Comment:
I really don't like this abbreviation of SDO, I sort of want to change everywhere 🤔

Reply:
I think better than StandardDirectionOrder everywhere to be honest :P